packages/form-field/src/FormField/FormField.styles.ts

```diff
@@ -30,9 +30,12 @@ export const getFontSize = ({
   }
 };
 
-export const labelTextContainerStyle = css`
+export const textContainerStyle = css`
   display: flex;
   flex-direction: column;
+`;
+
+export const paddingBottom = css`
```

Comment:
can we rename this `marginBottom` or `spacingBottom`? I don't like mixing up margin/padding since they can have different behavior